crates/runtime/src/instance/allocator/pooling.rs

```diff
@@ -259,12 +259,12 @@ pub enum PoolingAllocationStrategy {
 }
 
 impl Default for PoolingAllocationStrategy {
-    #[cfg(feature = "memfd-allocator")]
+    #[cfg(feature = "memfd")]
     fn default() -> Self {
         Self::ReuseAffinity
     }
 
-    #[cfg(not(feature = "memfd-allocator"))]
+    #[cfg(not(feature = "memfd"))]
```

Comment:
Could this use `cfg(memfd)` to have the right behavior on macOS/Windows with `memfd` enabled, and could this also be structured like:

```rust
if cfg!(memfd) {
    Self::ReuseAffinity
} else {
    Self::NextAvailable
}
```

to avoid conditional compilation?

Reply:
Ah yes, fixed, thanks!